scss/mixins/_screen-reader.scss

```diff
@@ -16,10 +15,9 @@
 
 // Use in conjunction with .sr-only to only display content when it's focused.
 //
-// Useful for "Skip to main content" links; see https://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G1
+// Useful for "Skip to main content" links; see [https://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G1](https://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G1)
```

Comment:
This kind of changes is what a Markdown lib could help :) Or we could change the script to look for `/https?/` or something so that links are automatically converted.